gradebookng/tool/src/java/org/sakaiproject/gradebookng/GradebookNgApplication.properties

```diff
@@ -227,6 +226,27 @@ importExport.selection.previewGrades.studentID.heading = Student ID
 importExport.selection.previewGrades.studentName.heading = Student Name
 importExport.selection.previewGrades.grade.heading = Grade
 
+importExport.error.grade = An error occurred importing a grade. Please check the file.
+importExport.error.comment = An error occurred importing a comment. Please check the file.
+importExport.commentname = + comments
+importExport.error.incorrecttype = The file you uploaded is not recognised as a CSV or Excel file.
```

Comment:
Is it possible to import from Excel?

Reply:
It sure is! It just wasn't advertised previously in GBNG. The message here refers to CSV being 'preferred' because we discovered some weird data/saving issues that can happen in Excel files.

Sometimes, Excel fails to save the exact floating point value as input by the user, so in the raw data behind the scenes, it can end up being a really weird number with a lot of decimal places. This ends up causing validation errors (or prior to this PR, it would just fail to import without telling the user why), because the grade is invalid with more than 2 decimal places. The user would never be aware of this, because in the UI Excel will show the value they input with correct decimal points. Hope I explained this well enough.

These issues don't affect CSV because it's just plain ol' text data.